CODEOWNERS

```
@@ -0,0 +1,5 @@
+# This file defines code ownership for the repository
+# Each line is a file pattern followed by one or more owners
+
+# Global owner for everything in the repository
+* @kyleryan207
```

File should end with a newline character for better Git compatibility and to follow POSIX standards.

```suggestion
* @kyleryan207
```